src/vuex/modules/networks.js

```diff
@@ -0,0 +1,88 @@
+import gql from "graphql-tag"
```

Comment:
I don't think we need this module. We can use just Apollo for storage and just store the `chain_id` in the session module. Each component that needs the Network info can request it.